library/std/src/process.rs

```diff
@@ -1491,6 +1491,66 @@ impl From<fs::File> for Stdio {
     }
 }
 
+#[stable(feature = "stdio_from_stdio", since = "1.58.0")]
```

Comment:
I think this magic constant is new since the PR was opened:
https://rustc-dev-guide.rust-lang.org/stabilization_guide.html#updating-the-feature-gate-listing

```suggestion
#[stable(feature = "stdio_from_stdio", since = "CURRENT_RUSTC_VERSION")]
```

Reply:
(This applies to all instances of the compiler version.)